### PR TITLE
1.8.2 added

### PR DESCRIPTION
**Pull request by zalun**
1.8 disabled

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -131,9 +131,9 @@
 XPI_AMO_PREFIX = "ftp://ftp.mozilla.org/pub/mozilla.org/addons/"
 
 # The lowest approved SDK available for add-ons
-DISABLED_SDKS = ('1.4', '1.4.1', '1.4.1-w-1', '1.4.2')
+DISABLED_SDKS = ('1.4', '1.4.1', '1.4.1-w-1', '1.4.2', '1.8')
```
Comment (seanmonstar) [inline settings.py:134]: Is there need to add it to the disabled list?

Won't changing `LOWEST_APPROVED_SDK` to 1.8.2 be enough?

Comment (zalun) [inline settings.py:134]: disabled is removing it entirely from the list.
the request was "deprecating/removing 1.8"

**Comment by zalun** [comment]
@seanmonstar please r+/- it

**Comment by seanmonstar** [comment]
r+

**Pull request closed without merging**